mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/146 - computer property panel

Closes #146 

To test:
- Run any scenario
- Open component view and select a computer
- Make sure property panel has correct information
